09. cli-and-http-request/clihelper/WriteUsingErrorLoggers.go

```diff
@@ -0,0 +1,10 @@
+package clihelper
+
+func WriteUsingErrortLoggers(cliBindingProperties CliBindingProperties, line string) {
+	_, err := w.Write(line)
```

Comment:
where is `w` defined?? we should omit this line

Reply:
w not part of this. sorry